oauth2client/contrib/gce.py

```diff
@@ -55,8 +52,7 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
```

Comment:
Yep, confirmed that list GCE instances with both ADC and directly instantiated `gce.AppAssertionCredentials`